### ISSUE TITLE
MacBook Pro 16" 2019 bug does not boot

### ISSUE THREAD

**Issue by ekultails** [ekultails/mac-linux-gaming-stick#79]
The GRUB menu shows up but then booting up the operating system fails complaining that the storage device cannot be found. I can confirm that the storage device for the USB drive (tested with a flash drive and external SSD) does not show up anymore.

This appears to be a kernel regression. Here is what I have tested so far:

- 5.4.114-1-MANJARO = works
- 5.4.124-1-MANJARO = does not work
- 5.10.32-1-MANJARO = works
- 5.10.41-1-MANJARO (Mac Linux Gaming Stick 2.0.0) = works
- 5.10.42-1-MANJARO = does not work
- 5.10.48-1-MANJARO (currently in the "testing" branch) = does not work

**Comment by ekultails** [comment]
Troubleshooting for this issue started here: https://github.com/ekultails/mac-linux-gaming-stick/issues/71

**Comment by ekultails** [comment]
I was able to recreate the known-working packages for Linux 5.10 from Mac Linux Gaming Stick 2.0.0.

```
$ yay -S fakepkg
$ fakepkg linux510
$ fakepkg linux510-headers
```

This has been confirmed to be working on a new build for 2.1.0-dev with this fix in via: f66d9dc41f2ff623c15f72c36c1cb09089d540e8
